src/main/java/com/google/sps/data/Section.java

```diff
@@ -14,20 +14,25 @@
 
 package com.google.sps.data;
 
+import java.util.Collections;
+import java.util.List;
+
 public class Section {
     private String professor;
-    private TimeRange[] meetingTimes;
-    /*
-    * meeting times is an array
-    * Where the 0th index is Sunday and the 6th is Saturday
-    * [_,_,_,_,_,_,_] no meeting times
-    * [_,*,_,*,_,*,_] class MWF
-    * [_,_,*,_,*,_,_] class TTh
-    * [*,_,_,_,_,_,*] class SatSun
-    */
-    public Section(String professor, TimeRange[] meetingTimes){
+    private List<TimeRange> meetingTimes;
+    // Meeting times will be sorted in ascending order of time, 
+    // with the first lesson being first in the list
```

Comment:
Comments should generally go above the code it's referring to.

Reply:
Resolved!